docs/config-files/config_v0.py

```diff
@@ -0,0 +1,50 @@
+# References:
+# https://pubs.acs.org/doi/suppl/10.1021/acs.analchem.8b00850/suppl_file/ac8b00850_si_001.pdf
+# https://docs.python.org/3/library/configparser.html
+# https://pubs.acs.org/doi/pdf/10.1021/acs.jchemed.7b00361
+# https://pubs.acs.org/doi/suppl/10.1021/acs.jchemed.7b00361/suppl_file/ed7b00361_si_001.pdf
+# https://pdfs.semanticscholar.org/5ac8/4dfb9170928fbf315daf9dab7bf7b07bb95e.pdf
+# https://www.researchgate.net/post/What_should_be_the_parameters_for_chronoamperometry_of_FeCl3_solution
+
+# config parameter 1: mode (this will be implemented as a dropdown menu)
+# selectable modes: 
+    # 1) linear sweep voltammetry - connect the reference and counter electrodes together 
+    # 2) chronoamperometry - 
+    # 3) cyclic voltammetry -
+
+# mode dependent paramters 
+# scan rate [mV/s]
+# the potential was varied linearly at the speed of the scan rate. This determines accuracy of cyclic voltammetry measurement
+
+
+# Example of a config file
+
+import configparser
+
+config = configparser.ConfigParser()
+
+config['DEFAULT'] = {'mode' : '0',
```

Comment:
something seems to have gone wrong here....your config has ended up in a python file again instead of a yaml.

Reply:
Sorry for my limited experience in using git. I thought that config_v0.py was deleted but it was actually included in the pull request. You can ignore this one.

Reply:
hahaha, no worries.

you can type ``git delete path/to/filename.ext`` to remove it from the project.